package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "typesafe-i18n",
-	"version": "5.24.3",
+	"version": "5.24.4",
```

Comment:
Please revert those changes. The version will be updated during the release workflow.

Reply:
Reverted